bundles/persistence/org.openhab.persistence.rrd4j/README.md

```diff
@@ -63,38 +66,57 @@ Sets the timeintervall(seconds) between consecutive readings.
+Each (step x steps) seconds (the step is globally defined for the RRD, 60s in our example) the most-left box is emptied, the content of all boxes is moved one box to the left and new content is added to the most right box.
 The "steps" value is defined per archive it is the third parameter in the archive definition.
 The number of boxes is defined as the fourth parameter.
 
 The purpose to have several archives is raised if a different granularity is needed while displaying data for different timespans.
-In the above examples data for each second are saved for the last hour (granularity 1), looking at the last four hours a granularity of 10 (i.e. 10 readings are consolidated to one reading) is used and so forth.
+In the example below data for each minute are saved for the last 8 hours (granularity 1), looking at the last 24 hours a granularity of 10 (i.e. 10 readings are consolidated to one reading) is used and so forth.
 For the first archive (and maybe the only one) a steps-size of one should be used.
 This way a sample is taken after each step.
-In this special case the selection of the consolidation function is of no effect (a single reading is equal to the MAX, MIN, AVERAGE and LAST of this reading).
   
 ### Example
 
-So in the above examples, we have 480 boxes, which each represent the value of one minute (Step is set to 60s, Granularity = 1).
-If more than one value is added to the database within (step x step) seconds (and thus more than one value would be stored in one box), the "consolidation function" is used.
+So in the example shown below, we have 480 boxes, which each represent the value of one minute (Step is set to 60s, Granularity = 1).
+If more than one value is added to the database within (step x steps) seconds (and thus more than one value would be stored in one box), the "consolidation function" is used.
 openHAB uses AVERAGE as default for numeric values, so if you add 20 and 21 within one minute, 20.5 would be stored.
 480 minutes is 8 hours, so we have a 8h with the granularity of one minute.
 
-The same goes for the next archives, for larger time spans, the stored values are less "exact". However, usually you are not interested in the exact temperature for a selected minute two years ago.
+The next archive has 144 boxes, which each represent the value of ten minutes (Step is set to 60s, Granularity = 10).
+1440 minutes is 24 hours, so we have a full day with with the granularity of 10 minutes.
+
+The same goes for following archives, for larger time spans, the stored values are less "exact". 
+However, usually you are not interested in the exact values for a selected minute some time ago.
+
+Note that the period covered for each archive starts with the actual time backwards! 
+No value is stored beyond the timeframe covered by the largest archive!
```

Comment:
Delete.

Reply:
Please confirm only the sentence "No value is stored.." is to be deleted.

Reply:
Yes. Why would anyone expect data to be stored beyond the requested range?